ewapt.py

```diff
@@ -0,0 +1,1216 @@
+            usermodel.change_slimecoin(n=aptmodel.rent * -4, coinsource=ewcfg.coinsource_spending)
+            aptmodel.rent = 0
+            aptmodel.poi = ""
+            aptmodel.apt_class = ewcfg.property_class_c
+            usermodel.persist()
+            aptmodel.persist()
+
+            await toss_squatters(cmd.message.author.id, cmd.message.server.id)
+    return await ewutils.send_message(cmd.client, cmd.message.channel, ewutils.formatMessage(cmd.message.author, response))
+
+
+async def toss_squatters(user_id = None, server_id = None):
+    player_info = EwPlayer(id_user=user_id)
+    if player_info.id_server != None:
+        try:
+            conn_info = ewutils.databaseConnect()
+            conn = conn_info.get('conn')
+            cursor = conn.cursor();
+            client = ewutils.get_client()
+
+            # get all players visiting an evicted apartment and kick them out
+            cursor.execute(
+                "SELECT {} FROM users WHERE {} = %s AND {} = %s".format(
+                    ewcfg.col_id_user,
+                    ewcfg.col_visiting,
+                    ewcfg.col_id_server
+                ), (
+                    player_info.id_user,
+                    server_id
+                ))
+
+            squatters = cursor.fetchall()
+
+            for squatter in squatters:
+                sqt_data = EwUser(id_user=squatter[0], id_server=player_info.id_server)
+                server = ewcfg.server_list[sqt_data.id_server]
+                member_object = server.get_member(squatter[0])
+                sqt_data.poi = sqt_data.poi[3:]
+                sqt_data.visiting = ewcfg.location_id_empty
+                sqt_data.persist()
+                await ewrolemgr.updateRoles(client=client, member=member_object)
+        finally:
+            # Clean up the database handles.
+            cursor.close()
+            ewutils.databaseClose(conn_info)
+
+
+def toss_items(id_user = None, id_server = None, poi = None):
+    if id_user != None and id_server != None and poi != None:
+        inv_toss = ewitem.inventory(id_user=id_user, id_server=id_server)
+        for stuff in inv_toss:  # toss all items out
+            stuffing = ewitem.EwItem(id_item=stuff.get('id_item'))
+            stuffing.id_owner = poi.id_poi
+            if stuff.get('item_type') == ewcfg.it_food and id_user[-6:] == ewcfg.compartment_id_fridge:
+                stuffing.item_props['time_expir'] = str(int(float(stuffing.item_props.get('time_expir'))) + (int(time.time()) - int(float(stuffing.item_props.get('time_fridged')))))
+                stuffing.item_props['time_fridged'] = '0'
+            stuffing.persist()
+
+
+def letter_up(letter = None):
+    if letter == ewcfg.property_class_a:
+        return ewcfg.property_class_s
+    elif letter == ewcfg.property_class_b:
+        return ewcfg.property_class_a
+    elif letter == ewcfg.property_class_c:
+        return ewcfg.property_class_b
+
+
+def get_item_str_name(item_sought = None):
```

Comment:
all dicts returned by ewitem.find_item() have a 'name' attribute that already serves this purpose